package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "node-ext2fs",
+  "name": "ext2fs",
   "version": "0.0.2",
   "description": "NodeJS native bindings to the libext2fs for cross-platform ext filesystem handling",
   "author": "Petros Angelatos <petrosagg@resin.io>",
   "license": "Apache-2.0",
   "main": "index.js",
+  "gypfile": true,
   "scripts": {
     "configure": "node-gyp configure",
     "build": "node-gyp build",
     "test": "mocha",
     "publish-please": "publish-please",
-    "prepublish": "publish-please guard"
+    "prepublish": "publish-please guard",
+    "install": "node-gyp rebuild"
```

Comment:
It runs `clean`, `configure` and `build`, and is usually what you'd want for dependants installing this (so that it cleans & reconfigures when it's updated)